p4-fusion/main.cc

```diff
@@ -50,13 +49,14 @@ int Main(int argc, char** argv)
 	const bool noMerge = arguments.GetNoMerge();
 	const std::string depotPath = arguments.GetDepotPath();
 	const std::string srcPath = arguments.GetSourcePath();
+	FileData::repoPath = srcPath;
```

Comment:
TODO: Dirty hack.

Reply:
What does this hack do?

Reply:
it got the repoPath to where I needed it using a static field, but I fixed it by keeping it git_api.h local now